packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -115,28 +115,61 @@ interface OrganizationMongoSelectProps extends BaseMongoSelectProps {
   isOnPrem?: boolean;
 }
 
-const onKeyDown: React.KeyboardEventHandler = e => {
```

Comment:
You can  definitely update this function so you can reuse it across both selectors. Seems important to guarantee the similarity in behavior.

Reply:
👍